test/borrower/feed/navfeedPV.t.sol

```diff
@@ -200,15 +207,41 @@ contract NFTFeedTest is DSTest, Math {
         uint256 riskGroup = 1;
         uint256 value = 100 ether;
         uint256 loan = 1;
-        uint256 debt = 40 ether;
         uint256 maxCeiling = 70 ether;
 
         feed.update(nftID, value, riskGroup);
 
         shelf.setReturn("shelf", address(1), 1);
         assertEq(feed.ceiling(loan), maxCeiling);
-
         feed.borrow(loan, maxCeiling);
+        assertEq(feed.currentNAV(), maxCeiling);
+        assertEq(feed.latestNAV(), maxCeiling);
+    }
+
+    function testFailBorrowAfterWriteOff() public {
+        bytes32 nftID = feed.nftID(address(1), 1);
+        uint256 riskGroup = 1;
+        uint256 value = 100 ether;
+        uint256 loan = 1;
+        uint256 maxCeiling = 70 ether;
+        uint256 borrowAmount = 20 ether;
+
+        feed.update(nftID, value, riskGroup);
+
+        shelf.setReturn("shelf", address(1), 1);
+        assertEq(feed.ceiling(loan), maxCeiling);
+        feed.borrow(loan, borrowAmount);
+        pile.incDebt(loan, borrowAmount);
+        assertEq(feed.currentNAV(), borrowAmount);
+        assertEq(feed.latestNAV(), borrowAmount);
+
+        feed.writeOff(loan);
+        assertEq(pile.values_uint("changeRate_loan"), loan);
+        assertEq(pile.values_uint("changeRate_rate"), feed.WRITEOFF_RATE_GROUP());
+        assertEq(feed.currentNAV(), 0);
+        assertEq(feed.latestNAV(), 0);
+        assert(feed.zeroPV(loan));
+        feed.borrow(loan, borrowAmount); // try to borrow again
```

Comment:
Just a comment that we don't need to address now, but for the future: with Foundry, you can use `vm.expectRevert` instead of `testFail`. This way you can check that a specific revert happens, rather than that one of the calls in the entire test fails.